### ISSUE TITLE
SendMailJob always runs on Default Queue

### ISSUE THREAD

**Issue by goaround** [spatie/mailcoach-support#281]
Hi,

since my upgrade to v3 the SendMailJob always runs on the `default` Queue. I think it should run on the `send-mail` queue but I can't figure out why this happens. 

My `perform_on_queue` looks fine and in the Job data `queue is` set to `send-mail` but it still runs on the default Queue.

```
'perform_on_queue' => [
        'calculate_statistics_job' => 'mailcoach',
        'send_campaign_job' => 'send-campaign',
        'send_mail_job' => 'send-mail',
        'send_test_mail_job' => 'mailcoach',
        'send_welcome_mail_job' => 'mailcoach',
        'process_feedback_job' => 'mailcoach-feedback',
        'import_subscribers_job' => 'mailcoach',
    ],
```

![grafik](https://user-images.githubusercontent.com/5927337/100000950-a7f9bf80-2dc2-11eb-811e-7de8381005a7.png)

![grafik](https://user-images.githubusercontent.com/5927337/100000315-b693a700-2dc1-11eb-9786-a1c0548afb12.png)

Any idea how this can happen?

**Comment by riasvdv** [comment]
This should have been recently fixed in laravel-mailcoach 3.2.13, can you confirm you're on that version or a later one?

**Comment by goaround** [comment]
Shame on me, no... Thank you!
